classes/models.py

```diff
@@ -3,14 +3,49 @@
 
 # Create your models here.
 
-# Print helpers
-print_format = "{:<15}" * 2
+# Proposed changes:
+"""
+Split a class into two models - Class and Section
+    - Can differentiate between REC/LEC/etc more
+      concretely. Have a model for each kind
+    
+    - Keep historical data. Each time a new class
+      list is released, simply append a section
+      to an existing class instead of deleting
+      old classes. This plays well with the FCQ
+      stuff as well
+
+    - Class Model (Generic for all types)
+        department (Foreign key to department object)
+        course_subject
+        
+    
+    - Section Model (One for each type)
+        class (Foreign key to the actual class object)
+        credits
+        section number
+        start_time
+        end_time
+        instructor (foreign key to instructor)
+        days
+        session
+        semester (Spring/Fall/Summer <year>)
+        max_enrollment
+        building_room
+        campus
+        schedule (Many to Many Field)
+"""
```

Comment:
As I understand, Classes are more persistent (i.e. we use the same class that holds CSCI 2270 for multiple semesters), and then sections hold the actual data for the semester? I could see that being a good change

Also one small thought on the class search functionality, I think the lowest value section number should show up on the search page, that way the user is taken first to the 'most important' section. (I find I'm occasionally brought to recitations first and I'm most interested in seeing the lecture times)